code/modules/client/preference_setup/loadout/loadout.dm

```diff
@@ -275,12 +283,15 @@ var/list/gear_datums = list()
 				temp_html += " <a href='?src=\ref[src];gear=[G.display_name];tweak=\ref[tweak]'>[tweak.get_contents(get_tweak_metadata(G, tweak))]</a>"
 			temp_html += "</td></tr>"
 
-		if(ticked)
-			ticked_items_html += temp_html
-		else if(!available)
+		if(G.always_use_available_items_list)
```

Comment:
I don't like the headers and this, in how it circumvents the existing system of {selected items at the top}, {available items in the middle}, and {unavailable at the bottom}.

Meaning that items that are `always_use_available_items_list` will always be at the top, even if they can't be selected and are unavailable. Potentially makes the loadout even more of a mess than it is currently.

Reply:
What would the headers be even used for, that can't already be done with adding a new category.